bot/bot_commands.py

```diff
@@ -493,6 +500,25 @@ async def set_unicode_preference(interaction: discord.Interaction, use_unicode:
             title="Preference Updated",
             description=data["message"])
 
+    @bot.tree.command(name=Command.get("roll").name, description=Command.get("roll").description)
+    async def roll(interaction: discord.Interaction):
+        await interaction.response.defer()
+        user = interaction.user
+
+        volcano_number = random.randint(1, config.ROLL_WINNING_NUMBER)
+        if volcano_number == config.ROLL_WINNING_NUMBER:
+            await send_success_message(
+                interaction,
+                title=f"WINNER! You rolled {volcano_number}!",
+                description=f"Congratulations {user.mention}!\nYou won the {config.ROLL_COMMAND} game!\n"
+            )
+        else:
+            await send_info_message(
+                interaction,
+                title=f"You rolled {volcano_number}.",
+                description=f"Sorry {user.mention}, that's not a winning number.\nBetter luck next time!"
+            )
```

Comment:
```suggestion
        won = (volcano_number == config.ROLL_WINNING_NUMBER)
        title_content = f"{'WINNER' if won else 'LOSER'}! You rolled {volcano_number}!"
        if won:
            await send_success_message(
                interaction,
                title=title_content,
                description=f"Congratulations {user.mention}!\nYou won the {config.ROLL_COMMAND} game!\n"
            )
        else:
            await send_info_message(
                interaction,
                title=title_content,
                description=f"Sorry {user.mention}, that's not a winning number.\nBetter luck next time!"
            )
```
reduce duplicated text like this

Reply:
Seems unnecessary